examples/keyvalue/server/main.go

```diff
@@ -23,15 +23,11 @@ package main
 import (
 	"log"
 
-	"go.uber.org/fx/modules/yarpc"
 	"go.uber.org/fx/service"
 )
 
 func main() {
-	svc, err := service.WithModule(
-		yarpc.New(yarpc.ServiceCreateFunc(NewYARPCThriftHandler)),
-		service.WithRole("service"),
-	).WithOptions(
+	svc, err := service.WithModule(nil).WithOptions(
```

Comment:
Yeah but we have version contol. We can easily bring it back, lets not leave the stuff around in case we might want to use it sometime soon.

It's the same reason we don't comment out a whole file but leave it checked in, right?